pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -249,6 +223,13 @@ let
         ];
       });
 
+
+      fastapi = super.fastapi.overridePythonAttrs (old: rec {
+        # Flaky test:
+        #     ResourceWarning: Unclosed <MemoryObjectSendStream>
+        doCheck = false;
+      });
```

Comment:
It seems like it’d be better to do this in the actual `fastapi` if we want to do it, rather than hacking around it for a specific downstream package?

Reply:
I do not know whether it's flaky in general or only in this overridden package set. I've added a comment for that now.

I only observed it in this package set, not sure whether that justifies disabling tests also for normal `fastapi`.